### PR TITLE
Added player name page and removed write test button

### PR DESCRIPTION
**Pull request by mathewantony31**
Updated the flow so that you enter your player name in a separate screen before seeing the Host/Join options. I also removed the test button to manually change state to write (it's only host-controlled now!)

### CODE REVIEW

File: web/html/index.html [hunk 1]
```diff
@@ -1,9 +1,10 @@
 <html>
     <head>
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/css/bootstrap.min.css" integrity="sha384-/Y6pD6FV/Vv2HJnA6t+vslU6fwYXjCFtcEpHbNJ0lyAFsXTsjBbfaDjzALeQsN6M" crossorigin="anonymous">
+        <link type="text/css" rel="stylesheet" href="./style.css">
```
Comment (philkjacobs) [inline web/html/index.html]: should be something like web/static/css/style.css

File: web/js/main.jsx [hunk 2]
```diff
@@ -137,6 +136,7 @@ class Application extends React.Component {
   }
 
   joinGameButtonPressed(){
+    console.log("ALSO WORKS!")
```
Comment (philkjacobs) [inline web/js/main.jsx]: 🤔

File: web/js/main.jsx [hunk 3]
```diff
@@ -172,16 +172,26 @@ class Application extends React.Component {
 
     // Handle messages sent by the server.
     socket.onmessage = function(event) {
+      console.log("RECEIVED MESSAGE FROM SERVER")
```
Comment (philkjacobs) [inline web/js/main.jsx]: 🤔

File: web/js/main.jsx [hunk 4]
```diff
@@ -225,7 +235,7 @@ class Application extends React.Component {
           })
           console.log("Supers to be read are "+this.state.supers)
         }
-      }
+    }
```
Comment (philkjacobs) [inline web/js/main.jsx]: spacing was correct before

**Review by philkjacobs — CHANGES_REQUESTED** [review]

**Comment by mathewantony31** [comment]
@philkjacobs changes made! i'll fix the css in a separate diff

**Comment by mathewantony31** [comment]
Finally fixed css issue and added css to the first three pages (Enter name, Menu, waiting room). Right now, it only looks okay if you change your browser to a mobile device.

**Comment by mathewantony31** [comment]
Added a "Copy" modal to copy the game ID to clipboard for easy sharing. Also added basic game css to all screens.

**Review by philkjacobs — APPROVED** [review]

**Pull request merged**